internal/store/postgres/migrations/000001_create_providers_table.down.sql

```diff
@@ -0,0 +1,3 @@
+DROP TABLE IF EXISTS "providers";
```

Comment:
do we just adding a migration scripts now?
if yes I think this migration could be on separate PR so it will be in the separate commit in the main branch

Reply:
To decouple DB migration changes from app logic changes. Commit better to be as granular as possible and I see DB migration and app logic as two different changes. Both needs to be tested differently. If we need to revert, it is easier to do it if DB migration has its own commit.

Reply:
we can decouple gorm to go-migrate migration replacement into separate PR, but even reverting that commit in the future would cause an error after this "permissions" column changes applied because this changes depends on go-migrate

Reply:
As discussed in this [comment](https://github.com/odpf/guardian/pull/220#discussion_r927373822). This PR could just have a migration file `000005_add_permissions_to_appeal.down.sql`. The version 1-4 could be in a separate PR and being merge first.